### PR TITLE
Add post on CVE-2023-44487

### PR DESCRIPTION
**Pull request by kkraune**
... or @frodelu

### CODE REVIEW

File: _posts/2023-10-11-cve-2023-44487.md [hunk 1]
```diff
@@ -0,0 +1,27 @@
...
+    Vespa-8.240.5 is available with updated dependencies to address CVE-2023-44487.
+---
+
+2023-10-10, details of the vulnerability now named HTTP/2 Rapid Reset
+([CVE-2023-44487](https://nvd.nist.gov/vuln/detail/CVE-2023-44487))
+were announced as a vulnerability impacting most (if not all) HTTP/2 servers in the industry.
+In the Vespa container, we bundle the Jetty HTTP server.
+
+Being given an early heads up about this, we were ready to release an update version of Vespa (8.240.5)
+that included Jetty's patch for this vulnerability as soon as it was released.
```
Comment (bjorncs) [inline _posts/2023-10-11-cve-2023-44487.md:18]: Should we call it a patch or just a release? 
Link to https://github.com/eclipse/jetty.project/releases/tag/jetty-11.0.17.

File: _posts/2023-10-11-cve-2023-44487.md [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+excerpt: >
+    Vespa-8.240.5 is available with updated dependencies to address CVE-2023-44487.
+---
+
+2023-10-10, details of the vulnerability now named HTTP/2 Rapid Reset
+([CVE-2023-44487](https://nvd.nist.gov/vuln/detail/CVE-2023-44487))
+were announced as a vulnerability impacting most (if not all) HTTP/2 servers in the industry.
+In the Vespa container, we bundle the Jetty HTTP server.
+
+Being given an early heads up about this, we were ready to release an update version of Vespa (8.240.5)
```
Comment (bjorncs) [inline _posts/2023-10-11-cve-2023-44487.md:17]: Consider adding a timeline with when Jetty release was available, included in our build, release to Cloud, etc.

File: _posts/2023-10-11-cve-2023-44487.md (hunk 1 above)
Comment (kkraune) [reply _posts/2023-10-11-cve-2023-44487.md:18]: will change to release, good point

File: _posts/2023-10-11-cve-2023-44487.md [hunk 3]
```diff
@@ -0,0 +1,27 @@
...
+that included Jetty's patch for this vulnerability as soon as it was released.
+
+This version has been rolled out in the Vespa Cloud, within the applications' block-windows.
```
Comment (bratseth) [inline _posts/2023-10-11-cve-2023-44487.md]:
```suggestion
If you are using Vespa Cloud, no action is needed as you have already been upgraded to the safe release.
```

File: _posts/2023-10-11-cve-2023-44487.md (hunk 2 above)
Comment (kkraune) [reply _posts/2023-10-11-cve-2023-44487.md]: added more details

File: _posts/2023-10-11-cve-2023-44487.md [hunk 4]
```diff
@@ -0,0 +1,27 @@
...
+8.240.5 is also available at [hub.docker.com/r/vespaengine/vespa/tags](https://hub.docker.com/r/vespaengine/vespa/tags) for self-hosted usage.
```
Comment (bratseth) [inline _posts/2023-10-11-cve-2023-44487.md]:
```suggestion
If you are self hosting you are advised to upgrade to Vespa 8.240.5 as soon as possible.
```

File: _posts/2023-10-11-cve-2023-44487.md [hunk 5]
```diff
@@ -0,0 +1,27 @@
...
+date: '2023-10-11'
+skipimage: true
+tags: []
+excerpt: >
+    Vespa-8.240.5 is available with updated dependencies to address CVE-2023-44487.
+---
+
+2023-10-10, details of the vulnerability now named HTTP/2 Rapid Reset
+([CVE-2023-44487](https://nvd.nist.gov/vuln/detail/CVE-2023-44487))
+were announced as a vulnerability impacting most (if not all) HTTP/2 servers in the industry.
```
Comment (bratseth) [inline _posts/2023-10-11-cve-2023-44487.md:14]:
```suggestion
were announced. This vulnerability impacts most HTTP/2 servers in the industry,
```

File: _posts/2023-10-11-cve-2023-44487.md [hunk 6]
```diff
@@ -0,0 +1,27 @@
...
+In the Vespa container, we bundle the Jetty HTTP server.
```
Comment (bratseth) [inline _posts/2023-10-11-cve-2023-44487.md:15]:
```suggestion
including Vespa by embedding Jetty.
```

**Review by bjorncs — APPROVED** [review]

**Comment by kkraune** [comment]
sorry jon clicked to sooon will update

**Pull request merged**